test/constants.ts

```diff
@@ -10,22 +10,26 @@ export const TESTS_TIMEOUT = 30_000;
 export const SLEEP_FOR_RESULT = 3_000;
 
 // Addresses
-export const SECURITY_MODULE = '0xe57025E250275cA56f92d76660DEcfc490C7E79A';
+export const SECURITY_MODULE = '0xC77F8768774E1c9244BEed705C4354f2113CFc09';
```

Comment:
Let's specify which network it is and which devnet? We can specify it as
`holesky-0x...(locator address)`